opencti-platform/opencti-front/src/private/components/data/tasks/TasksList.jsx

```diff
@@ -222,6 +254,29 @@ const TasksList = ({ data }) => {
         } else if (task.task_ids) {
           listIds = truncate(R.join(', ', task.task_ids), 60);
         }
+        const lastTaskExecutionDate = task.work ? task.work.completed_time : task.last_execution_date;
+        const taskWorkProcessedNumber = task.work?.tracking?.import_processed_number ?? 0;
+        const taskWorkExpectedNumber = task.work?.tracking?.import_expected_number ?? 0;
+        const progressNumberDisplay = task.work ? ` ${taskWorkProcessedNumber}/${taskWorkExpectedNumber}` : '';
+        const provisioningNumberDisplay = task.work && (task.work.status === 'wait' || task.work.status === 'progress')
+          ? ` (Provisioning: ${task.task_processed_number}/${task.task_processed_number})`
+          : '';
+        const progressFullText = `${t_i18n('Progress')}${progressNumberDisplay}${provisioningNumberDisplay}`;
+        let progressValue = 0;
+        if (task.work) {
+          if (task.work.status === 'complete') {
```

Comment:
is it possible to use enum or constants to avoid having a string for status checks ?